pkg/controller/node/nodecontroller.go

```diff
@@ -449,7 +449,8 @@ func (nc *NodeController) monitorNodeStatus() error {
 			continue
 		}
 		node := nodeCopy.(*v1.Node)
-		for rep := 0; rep < nodeStatusUpdateRetry; rep++ {
+		var rep int
+		for rep = 0; rep < nodeStatusUpdateRetry; rep++ {
```

Comment:
Can you actually update this entire block to use `err = pkg/util/wait.PollImmediate(retrySleepTime, retrySleepTime*nodeStatusUpdateRetry, func(){ current loop content}`?  They probably want a time based boundary and perhaps even jitter.